api/internal/handler/ssl/ssl.go

```diff
@@ -147,6 +147,7 @@ func (h *Handler) Create(c droplet.Context) (interface{}, error) {
 	}
 
 	ssl.ID = input.ID
+	ssl.Status = 1
```

Comment:
Is this the cause of the bug?  Can you explain the reason?

Reply:
yes, it's the cause.
That is because in Go if does not explicitly assign a value, it will automatically assign a 0 value. So the SSL is disable.

Reply:
that is the magic number. we should use a `const` variable for a better name.

Reply:
and I think we'd better add some comment here, it should be useful for code reader

Reply:
OK.